upload/button_helper.py

```diff
@@ -43,7 +43,7 @@ def get_buttons_for_obj(
         usr = self.request.user
         url_name = self.request.resolver_match.url_name
 
-        if obj.status == choices.PS_VALIDATED_WITH_ERRORS and ph.user_can_finish_deposit(usr, obj) and url_name == 'upload_package_modeladmin_inspect':
+        if obj.status == choices.PS_READY_TO_BE_FINISHED and ph.user_can_finish_deposit(usr, obj) and url_name == 'upload_package_modeladmin_inspect':
```

Comment:
foi melhoria no nome do status ou é um status diferente
Melhor adicionar comentário

Reply:
@robertatakenaka Neste caso é um status diferente. Como a história pede que o usuário consiga ver o botão finalizar apenas após um pacote não ter erros de validação cuja ações sejam "to-fix", precisei criar esse outro estado, que é prévio a QA. Havendo "to-fix", o estado do pacote é PENDING_CORRECTION.